packages/codegen-ui/lib/generate-form-definition/generate-form-definition.ts

```diff
@@ -69,8 +69,8 @@ export function generateFormDefinition({
     .concat(
       Object.entries(form.sectionalElements).map(([elementName, elementConfig]) => ({
         name: elementName,
-        position: elementConfig.position,
-        excluded: false,
+        position: 'position' in elementConfig ? elementConfig.position : undefined,
+        excluded: 'excluded' in elementConfig ? elementConfig.excluded : false,
```

Comment:
I'm not sure if we should be stricter about the presence of the excluded flag. Ideally, it should only ever be present if the element actually is excluded. That could be confusing to customers though so let's just keep it as is.